compiler/resolution/preFold.cpp

```diff
@@ -267,9 +267,15 @@ static void setRecordCopyableFlags(AggregateType* at) {
       }
 
     } else {
-      // Try resolving a test init= to set the flags
-      const char* err = NULL;
-      FnSymbol* initEq = findCopyInitFn(at, err);
+      // TODO Jade:
```

Comment:
Let's throw a date on this as well, for extra wincing if we don't get to it right away :)